index.bs

```diff
@@ -1,11 +1,62 @@
-<pre class='metadata'>
-Title: Web Extensions
-Shortname: webextensions
-Level: 1
-Status: w3c/CG-DRAFT
-Group: WECG
-URL: https://w3c.github.io/webextensions
-Editor: Mukul Purohit, Microsoft Corporation https://www.microsoft.com, mpurohit@microsoft.com
-Editor: Tomislav Jovanovic, Mozilla https://www.mozilla.org/, tjovanovic@mozilla.com
-Abstract: [Placeholder] Abstract.
-</pre>
+<pre class='metadata'>
+Title: Web Extensions
+Shortname: webextensions
+Level: 1
+Status: w3c/CG-DRAFT
+Group: WECG
+URL: https://w3c.github.io/webextensions
+Editor: Mukul Purohit, Microsoft Corporation https://www.microsoft.com, mpurohit@microsoft.com
+Editor: Tomislav Jovanovic, Mozilla https://www.mozilla.org/, tjovanovic@mozilla.com
+Abstract: [Placeholder] Abstract.
+Markup Shorthands: markdown yes
+</pre>
+
+# Manifest
+
+A WebExtension must have a manifest file at its root directory.
+
+## Manifest file
+
+A manifest file is a [[!JSON]] document named `manifest.json`. Malformed JSON files are not supported.
+
+## Manifest keys
+
+If manifest keys that are not defined in this specification are specified, implementors may ignore those keys.
+
+If manifest keys that are defined in this specification are specified with a different JSON type than defined in this specification, implementors may ignore those keys.
+
+A manifest file must have these keys properly specified:
+
+* <a href="#key-name">`name`</a>: required.
```

Comment:
I think we should follow https://datatracker.ietf.org/doc/html/rfc2119 for words like MAY REQUIRE MUST

Reply:
I deliberately used a shorter formulation so that someone writing a WebExtensions can have a clear overview, with the formal definitions in the sections themselves. I also didn’t want to repeat myself.

Reply:
@Jack-Works, I don't quite follow. What's wrong with this block? It appears to be following the guidance in 2119.

Reply:
ah, I never realized it also defined «required» and «optional» also.